x/qgb/handler_test.go

```diff
@@ -119,3 +119,48 @@ func TestMsgValsetConfirm(t *testing.T) {
 	_, err = h(ctx, msg)
 	require.NoError(t, err)
 }
+
```

Comment:
This will be fixed in a subsequent PR after I write an issue about a confusion we have with the address